auction-app-backend/src/main/java/com/example/auctionapp/controller/AuthController.java

```diff
@@ -37,12 +39,12 @@ public AuthController(final AuthService authService) {
     }
 
     @PostMapping("/register")
-    public User register(@RequestBody final UserRequest user) {
+    public User register(@Valid @RequestBody final UserRequest user) {
         return authService.signUp(user);
     }
 
     @PostMapping("/login")
-    public JwtResponse login(@RequestBody final LoginRequest loginRequest, HttpServletResponse response) {
+    public JwtResponse login(@Valid @RequestBody final LoginRequest loginRequest, HttpServletResponse response) {
```

Comment:
I'm guessing you used this, to validated users input and in LoginRequest class you defined rules. 
If is something else, please write in response